deployment/ansible/roles/model-my-watershed.postgresql/tasks/main.yml

```diff
@@ -1,11 +1,4 @@
 ---
-- name: Configure PostgreSQL service definition
```

Comment:
This template is no longer necessary because we don't need to emit a custom Upstart event for other services to subscribe to.